utl/flatbuffer/vw_to_flat.cc

```diff
@@ -0,0 +1,294 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#include <sys/timeb.h>
+#include <fstream>
+#include <vector>
+
+#include "vw_to_flat.h"
+#include "options.h"
+#include "parse_args.h"
+#include "parse_regressor.h"
+#include "accumulate.h"
+#include "best_constant.h"
+#include "vw_exception.h"
+#include "options_boost_po.h"
+
+std::string to_flat::get_label_string(label_type_t label_type)
```

Comment:
This should be a utility exposed in vw itself, no reason to keep it in here. It may infact already exist?

Reply:
From what I can see it doesn't already exist. I am tempted to remove this altogether as it is only used in the to_flatbuff utility for printing out the label type when we convert from txt to flatbuffer, rather than adding a method to every label parser.